lib/dialogs/import/importers.ts

```diff
@@ -26,4 +28,46 @@ const importers: Array<importer> = [
   },
 ];
 
-export default importers;
+export const getImporters = () => {
+  return importers;
+};
```

Comment:
we should be able to expose `importers` directly - was there an issue with that?

Reply:
This works fine, and I've changed it to this now. The reason I added the getter function was just because when using this it read weird to me with the repeated importers name like `importers.importers.forEach`. It is only used once so far though so not really an issue.

Reply:
ha yeah well that can be a valid reason to rename things.

fun fact, you can also rename an import when you import them, but I honestly can't think of a better name.

maybe something like `availableImporters` but I'm not sure that's any better.